pkg/api/v1beta1/awspcaissuer_types.go

```diff
@@ -36,7 +35,28 @@ type AWSPCAIssuerSpec struct {
 	Region string `json:"region,omitempty"`
 	// Needs to be specified if you want to authorize with AWS using an access and secret key
 	// +optional
-	SecretRef v1.SecretReference `json:"secretRef,omitempty"`
+	SecretRef SecretReference `json:"secretRef,omitempty"`
+}
+
+type SecretReference struct {
```

Comment:
My suggestion would be that this class composes the `SecretReference` type (https://pkg.go.dev/k8s.io/api/core/v1#SecretReference) and also is renamed.
```suggestion
type AWSCredentialsSecretReference struct {
	// Empty JSON tag is required to solve encountered struct field "" without JSON tag error.
	SecretReference `json:""`
```